### DISCUSSION TITLE
Collateral amount when build txn

### DISCUSSION THREAD

**Discussion by thaopt** [Python-Cardano/pycardano#365] [General]
- hi @cffls, i figure out that when i build transaction with lucid (https://lucid.spacebudz.io/docs/overview/about-lucid/) and pycardano the collateral amount is huge different,
- for the same txn condition that interact with a smart contract, when i using lucid, it only 1.4 Ada in collateral but when i try the same with pycardano it is 3.6 ada ~3 times. 
- I know when tx success, the collateral amount will be returned back. but if not i will lose more asset than another.
-  How can i deal with this.

**Comment by thaopt** [comment]
Any idea on this @nielstron ?

**Comment by nielstron** [comment]
Hi @thaopt 
in order to help with this please provide a concrete transaction, the code to build it and ideally the cbor of the lucid and the PyCardano transaction.

does the lucid transaction succeed to submit to the network? 1.4 ada is a very low amount of collateral

**Comment by thaopt** [comment]
https://preview.cardanoscan.io/transaction/574aa5e8ec4ef37bfd3d078228d63b29710a2b5796f696d3459cbe6e8136f0ef?tab=utxo
https://preview.cardanoscan.io/transaction/a81b654d8675bab1b7fdf14072b63359147d1eaa33dd3cc67529fe371c11cfbb?tab=utxo

I have two simillar txns, which interact with the same smart contracts, but the collateral is very very different ~3 times multi
